internal/wrapnix/wrapper.go

```diff
@@ -162,3 +166,20 @@ func createSymlinksForSupportDirs(projectDir string) error {
 	}
 	return nil
 }
+
+// Creates a symlink for devbox in .devbox/virtenv/.wrappers/bin
+// so that devbox can be available inside a pure shell
+func createDevboxSymlink(projectDir string) error {
+
+	// Get absolute path for where devbox is called
+	devboxPath, err := filepath.Abs(os.Args[0])
+	if err != nil {
+		return errors.Errorf("Failed to create devbox symlink. Devbox command won't be available inside the shell. Error: %v", err)
```

Comment:
use `errors.Wrap`, `errors.Wrapf` or `errors.WithStack` instead. `errors.Errorf` is designed to create a new error instead of propagating an existing one.